pootle/static/css/editor.css

```diff
@@ -609,7 +615,7 @@ td.translate-focus
 td.translate-full div.translate-original
 {
     width: 100%;
-    margin: 0.5em 0;
+    margin: 0.7em 0;
```

Comment:
I wasn't happy for 100% with that change. But yes, I needed to make proper margins for plurals.